File: typescript-client-sdk/src/sdk/models/shared/eventattendeecreate.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class EventAttendeeCreate extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=additional" })
  additional?: Record<string, any>;

  @SpeakeasyMetadata({ data: "json, name=associatedObjectType" })
  associatedObjectType: string;

  @SpeakeasyMetadata({ data: "json, name=associationId" })
  associationId?: string;

  @SpeakeasyMetadata({ data: "json, name=email" })
  email?: string;

  @SpeakeasyMetadata({ data: "json, name=eventId" })
  eventId: string;

  @SpeakeasyMetadata({ data: "json, name=status" })
  status?: string;
}

File: go-client-sdk/pkg/models/shared/eventattendeecreate.go
package shared

type EventAttendeeCreate struct {
	Additional           map[string]interface{} `json:"additional,omitempty"`
	AssociatedObjectType string                 `json:"associatedObjectType"`
	AssociationID        *string                `json:"associationId,omitempty"`
	Email                *string                `json:"email,omitempty"`
	EventID              string                 `json:"eventId"`
	Status               *string                `json:"status,omitempty"`
}
